microproxy/event/replay.py

```diff
@@ -33,18 +33,18 @@ def handle(self, event):
                 port=viewer_context.port,
                 config=self.config,
                 scheme=viewer_context.scheme,
-                interceptor=self.proxy_server.interceptor)
-            yield self.proxy_server.layer_manager.run_layers(
-                layer_context)
+                interceptor=get_interceptor(self.config))
+            yield layer_manager.run_layers(layer_context)
```

Comment:
I suggest that layer_manager still be a field of ReplayHandler
1. Testability
2. Readilibilty (We could find out that ReplayHandler will depends on layer_manager by just see its __init__ function, and know that it will use layer_manager to run_layers)